### ISSUE TITLE
Unable to name file from "save as"-window.

### ISSUE THREAD

**Issue by lens17** [ONLYOFFICE/DesktopEditors#554]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

When trying to save/save as a file, I am unable to change the name from e.g. "Book1.xlsx" to "calculation.xlsx". I have to save it as "Book1.xlsx" currently and from the File Manager/Explorer I have to rename the file.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

1) Open OO
2) Create new file
3) Select "Save as..."
4) Try to enter a name 

**What is the expected behavior?**

It would be much appreciated if I could name the file. 

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
Version: ONLYOFFICE Desktop Editors Version 6.1.0.90
OS: KDE Neon 5.20
Previous versions: I cannot tell, just recently switched to OO.

**Comment by matveevms** [comment]
Hi, @lens17
Are you using the snap version of the editors?

**Comment by lens17** [comment]
Yes, this is indeed the case. 
Please excuse the delayed response.

**Comment by matveevms** [comment]
Indeed, we are aware of this problem in the snap version and we are working on fixing it. Duplicate of this https://github.com/ONLYOFFICE/snap-desktopeditors/issues/33
